tasks/next.config.js

```diff
@@ -5,6 +5,9 @@ const nextConfig = {
   swcMinify: true,
   transpilePackages: ['@helpwave/common'],
   output: 'standalone',
+  images: {
+    domains: ['helpwave.de'],
```

Comment:
it is needed so next can use the `<Images>` tag otherwise it threw an error for me

Reply:
The `<Images>` tag uses optimizations for the host domain and other domains, which have to be named in the `next.config.js`, because boring avatars uses a compression on the SVGs we use we also need to allow for this which is considered dangerous by `next`